### DISCUSSION TITLE
Remote delayed start

### DISCUSSION THREAD

**Discussion by ap0ll0nas** [gopro/labs#219] [Feature Requests]
When a user presses the remote record, start recording a specified number of sec later. Applicable when the remote goes out of range and user wants to start recording when out of range. Use case is biking a certain section of a trail.

**Comment by dnewman-gpsw** [comment]
Normal Labs does change a core behavior, it is more "yes add..."  The problem here is the disabling and the delaying of record button. While it might be doable, I'm less convinced of its value, as recording too early is not nearly as problematic as recording too late.  I will think on it.

**Reply by ap0ll0nas** [reply]
If the remote in range recording can be implemented, this would indeed be of much less value.
